### DISCUSSION TITLE
How to get GitHub as a Official sponsor for university's event?

### DISCUSSION THREAD

**Discussion by Parmesh119** [community/community#137143] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

In my university, we are organizing one event and want to get GitHub as an Official Sponsor for this event.
How to do it?

**Comment by ebndev** [comment]
Hi @Parmesh119, thanks for being a part of the GitHub Community!

Our `Sponsors` category is meant for conversations about our program, GitHub Sponsors. For your inquiry, you can open a ticket on our [Support Portal](https://support.github.com/contact?tags=rr-general-technical) and they should be able to direct you to the proper channels.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
